npm/src/upload/uploadform/upload-form.ts

```diff
@@ -221,17 +151,14 @@ export class UploadForm {
       this.showUploadingRecordsPage()
       this.folderUploader(this.selectedFiles, uploadFilesInfo)
     } else {
-      this.successMessage?.setAttribute("hidden", "true")
-      this.warningMessages.incorrectItemSelectedMessage?.setAttribute(
-        "hidden",
-        "true"
-      )
-      this.warningMessages.submissionWithoutSelectionMessage?.removeAttribute(
-        "hidden"
-      )
-
-      this.warningMessages.submissionWithoutSelectionMessage?.focus()
       this.addSubmitListener() // Readd submit listener as we've set it to be removed after one form submission
```

Comment:
Spelling in comment: Read**d** submit ...

Reply:
Oh, I thought [it was](https://www.collinsdictionary.com/dictionary/english/readd). But, I'll change it.